### DISCUSSION TITLE
My email did not go through verification process completely-- HELP

### DISCUSSION THREAD

**Discussion by havan22** [community/community#136411] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

- I clicked on GitHub new  account link in the email.
- I input email and password and username, created account.
- After I created the account, the page Verify with tick mark showed up and that was it. No further email and the page did not proceed to confirm

My account is failing to login - invalid/ username- password and when reset password - it says the email is not verified.  I do not know how to proceed and rectify this stuck email.

**Comment by trandangtrungduc** [comment]
I have been in the same situation as you. The only thing you can do is reset your account password. If you have problems with the link sent via email, you can try some of the following:

1. Quit and restart your browser.
2. Check the spam folder or blocked list in your email.
3. Use another browser or use your mobile phone for the verification process.
4. Many people have the same problem like this and still don't know the cause, so you just have to believe in luck.
Note: the verification link expires after 24 hours.

Good luck!
